service_configuration_lib/spark_config.py

```diff
@@ -929,31 +911,44 @@ def _append_event_log_conf(
 
         if len(self.spark_srv_conf.items()) == 0:
             log.warning('spark_srv_conf is empty, disable event log')
+            spark_opts.update({"spark.eventLog.enabled": "false"})
```

Comment:
why?

Reply:
`spark.eventLog.enabled` is default to false, perhaps we don't need this line if srv_config is empty?

Reply:
I have added this to be explicit and not leave the state to the default behaviour, since the log clearly says we are disabling it. I can remove this if you feel strongly against it

Reply:
I see, both looks okay to me

Reply:
Maybe we can remove this whole section? I can't think of a reason on why to disable to event log hen spark-srv-conf is empty. I recall generally spark event log used to be disabled to avoid risk of leaking of secrets. Since that is not happening anymore, AFAIK, maybe we can remove this section?